Sources/OktaSQLiteStorage/README.md

```diff
@@ -1,9 +1,27 @@
 #  Okta SQLite Storage SDK 
 
-OktaSQLiteStorage is a lightweight wrapper on top of GRDB framework that allows easy db creation and handling migration scenarios
+OktaSQLiteStorage is a lightweight wrapper on top of GRDB framework that allows easy db creation and handling of migration scenarios
+
+## Getting started
+
+### CocoaPods
+
+This SDK is available through [CocoaPods](https://cocoapods.org). To install it, add the following line to your Podfile:
+
```

Comment:
NIT: Any `Getting started` instructions for folks who utilize Swift Package Manager? 😄